src/utils/getActualDataList.ts

```diff
@@ -0,0 +1,31 @@
+import dayjs, { extend } from 'dayjs';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+
+type DataListType<T> = Array<T>;
+type DataItem = Record<string, string | HTMLElement>;
+interface DataItemWithDate extends DataItem {
+  date: string; // 'YYYY-MM-DD'
+}
+
+export type ActualDataListProps = {
+  dataList: DataListType<DataItemWithDate>;
+  actualDelayInDays: number;
+  sorted?: boolean;
+};
+
+export const getActualDataList = ({
+  dataList,
+  actualDelayInDays,
+  sorted = true,
+}: ActualDataListProps): DataListType<DataItemWithDate> | [] => {
+  extend(customParseFormat);
```

Comment:
Probably good idea to keep all plugins `extend` in one place

![image](https://github.com/rolling-scopes/site/assets/114234698/8b82b358-2315-47e3-9c67-a350834eb15b)
![image](https://github.com/rolling-scopes/site/assets/114234698/0a354705-935b-488c-b26e-79eb0664a82c)

Reply:
Very good idea! done